exercises/concept/temperature/.docs/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+You are part of a firefighter organization made of international volunteers. Over the last few calls your team has wasted valuable time translating from the new American team member charged with estimating the temperature inside the burning building. To save time, and people, you decide to create a program to automatically convert to and from Fahrenheit and Celsius for your new teammate.
```

Comment:
This bit of content belongs in the `instructions.md` document (see https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-instructions-md). The contents of the `introduction.md` file (this file) are used to provide a minimal explanation of the concepts the student needs to know to finish the exercise. See https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md